subprojects/core-api/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -208,6 +211,24 @@
      */
     StartParameter getStartParameter();
 
+    /**
+     * Creates a {@code Provider} implementation based on the provided value.
+     *
+     * @param value The {@code java.util.concurrent.Callable} use to calculate the value.
+     * @return The provider. Never returns null.
+     * @throws org.gradle.api.InvalidUserDataException If the provided value is null.
+     * @see org.gradle.api.provider.ProviderFactory#provider(Callable)
+     * @since 6.8
+     */
+    <T> Provider<T> provider(Callable<T> value);
+
+    /**
+     * Provides access to methods to create various kinds of {@link Provider} instances.
+     *
+     * @since 6.8
+     */
+    ProviderFactory getProviders();
```

Comment:
:x: Newly introduced methods should be `@Incubating`

Reply:
Sounds good.  I just made the guess that because ProviderFactory wasn't incubating anymore that accessor functions for it wouldn't be either.  😅

Reply:
Sounds good.  I just made the guess that because ProviderFactory wasn't incubating anymore that accessor functions for it wouldn't be either.  😅